src/Storage/TSStorageManager.h

```diff
@@ -24,7 +24,7 @@ extern NSString *const TSUIDatabaseConnectionDidUpdateNotification;
 - (void)setupDatabase;
 - (void)deleteThreadsAndMessages;
 - (BOOL)databasePasswordAccessible;
-- (void)wipeSignalStorage;
+- (void)resetSignalStorage;
```

Comment:
Just a heads up that this is used in Signal-iOS. Generally if there are breaking changes in ssk, try to open the corresponding Signal-iOS PR that can run it simultaneously.